From 6ea9a1ee9367fb35acff1c08a0dc4213ff4687a0 Mon Sep 17 00:00:00 2001
From: Tomasz Maciej Nowak <tmn505@gmail.com>
Date: Tue, 9 Apr 2019 15:53:42 +0200
Subject: [PATCH] arm64: dts: marvell: armada-3720-espressobin: add ports
 phandle

Instead of referencing the whole mdio node, add ports phandle to adjust
port labels in dts for different hardware iterations of ESPRESSObin
boards.

Signed-off-by: Tomasz Maciej Nowak <tmn505@gmail.com>
---
 arch/arm64/boot/dts/marvell/armada-3720-espressobin.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/marvell/armada-3720-espressobin.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3720-espressobin.dts
@@ -147,7 +147,7 @@
 
 		dsa,member = <0 0>;
 
-		ports {
+		ports: ports {
 			#address-cells = <1>;
 			#size-cells = <0>;
 
